code/index.html

```diff
@@ -3,28 +3,193 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
-  </head>
+    <title>Plant Film Festival</title>
+    <link rel="icon" href="./montera-favicon.png">
+    <link rel="stylesheet" href="./style.css" />
+
+    <!-- Adding google fonts -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
```

Comment:
This is just a preference thing, but instead of adding so many google font links, you can combine all the fonts and have just one link.  This can be done on https://fonts.google.com/ by selecting styles and copying the generated link.